organisations/admin.py

```diff
@@ -198,6 +198,38 @@ class UserAdmin(DjangoUserAdmin):
     date_hierarchy = "date_joined"
     form = UserAdminForm
 
+    def save_form(self, request, form, change):
+        """
+        Given a ModelForm return an unsaved instance. ``change`` is True if
+        the object is being changed, and False if it's being added.
+        Overridden so the request could be combined with the model instance save
+        and we could write a message about a mail is being sent.
+        """
+
+        original_user = form.instance
+
+        should_notify = self._has_organisations_changed(
```

Comment:
maybe this variable name is a bit misleading if `should_notify = True` doesn't necessarily mean that there should actually be notification